### DISCUSSION TITLE
Is this still in active development?

### DISCUSSION THREAD

**Discussion by kershnerd** [blitz-js/blitz#4384] [Q&A]
The docs are for Blitz V1 and Next 13.  Are there documentation for Blitz v2 somewhere?

**Comment by tordans** [comment]
The docs at https://blitzjs.com/docs/usage-next-13#app-directory are all for V2 AFAIK and that links is the place to learn about RSC and Blitz.

**Comment by marko911** [comment]
I find it strange that there is no documentation on using the core modules. For example this is the default file generated when you start a new project:
```
const {api, getBlitzContext, useAuthenticatedBlitzContext, invoke,withBlitzAuth} = setupBlitzServer({
  plugins: [
    AuthServerPlugin({
      ...authConfig,
      storage: PrismaStorage(db),
      isAuthorized: simpleRolesIsAuthorized,
    }),
    RpcServerPlugin({}),
  ],
  logger: BlitzLogger({}),
})

export {api, getBlitzContext, useAuthenticatedBlitzContext, invoke,withBlitzAuth}
```


Where would I check to see what each of these exported objects actually do and how to use them?
